EIPS/eip-7496.md

```diff
@@ -25,15 +25,15 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Contracts implementing this EIP MUST include the events, getters, and setters as defined below, and MUST return `true` for [ERC-165](./eip-165.md) `supportsInterface` for `0x12345678(placeholder, to be set when finalized)`, the 4 byte `interfaceId` for this ERC.
 
-If the contract does not wish for others to modify their metadata, the setters MAY revert when called.
+The contract MUST implement permissions for setters so only privileged users can update traits and metadata.
```

Comment:
Why make this mandatory?

Reply:
otherwise anyone can update anyone's metadata traits?

Reply:
The decision on whether that's acceptable is up to the contract creator--not sure why we'd try to make that decision for them

Reply:
ok, will remove